app/(playground)/p/[agentId]/components/navigation-panel.tsx

```diff
@@ -181,8 +492,320 @@ export function Overview() {
+	{
+		type: "github.issue_comment.reply" satisfies GitHubNextAction,
+		label: "Comment on trigger issue",
+	},
+] as const;
+const mockEventDataList = [
+	{
+		type: "github.issue_comment.body",
+		label: "Body",
+	},
+];
+function GitHubIntegrationForm() {
+	const { setting, repositories, upsertGitHubIntegrationSettingAction } =
+		useGitHubIntegration();
+	const { popoverOpen, setPopoverOpen } = useTabValue();
+	const { graph } = useGraph();
+	const [callSign, setCallSign] = useState(setting?.callSign ?? "");
+	const [eventNodeMappings, setEventNodeMappings] = useState<
+		GitHubEventNodeMapping[]
+	>(setting?.eventNodeMappings ?? []);
+	const processedMappings = useMemo(
+		() =>
+			eventNodeMappings
+				.map((eventNodeMapping) => {
+					const node = graph.nodes.find(
+						(node) => node.id === eventNodeMapping.nodeId,
+					);
+					if (node === undefined) {
+						return null;
+					}
+					return {
+						...eventNodeMapping,
+						id: `${eventNodeMapping.event}-${eventNodeMapping.nodeId}`,
+						node,
+					};
+				})
+				.filter((eventNodeMapping) => eventNodeMapping !== null),
+		[eventNodeMappings, graph.nodes],
+	);
+	const [upsertGitHubIntegrationSettingActionResult, action, upserting] =
+		useActionState(upsertGitHubIntegrationSettingAction, null);
+	return (
+		<form
+			className="grid gap-[16px] overflow-y-auto mx-[-20px] px-[20px]"
+			action={action}
+		>
+			{upsertGitHubIntegrationSettingActionResult?.result === "error" && (
+				<p>{upsertGitHubIntegrationSettingActionResult.message}</p>
+			)}
+			<ContentPanelSection>
+				<ContentPanelSectionHeader title="Repository" />
+				<Select
+					name="repositoryFullName"
+					placeholder="Choose repository"
+					options={repositories.map((repository) => ({
+						value: repository.full_name,
+						label: repository.full_name,
+					}))}
+					defaultValue={setting?.repositoryFullName}
+				/>
+			</ContentPanelSection>
+
+			<ContentPanelSection>
+				<ContentPanelSectionHeader title="Trigger" />
+				<ContentPanelSectionFormField>
+					<Label htmlFor="event">Event</Label>
+					<Select
+						name="event"
+						defaultValue={setting?.event}
+						placeholder="Choose event"
+						options={integrationEventList.map((integrationEvent) => ({
+							value: integrationEvent.type,
+							label: integrationEvent.label,
+						}))}
+					/>
+
+					<ContentPanelSectionFormField>
+						<Label htmlFor="callSign">Call sign</Label>
+						<Input
+							type="text"
+							name="callSign"
+							id="callSign"
+							placeholder="Enter call sign"
+							className="w-full"
+							value={callSign}
+							onChange={(e) => setCallSign(e.target.value)}
+						/>
+						<span className="text-black-70 text-[12px]">
+							You can call this agent by commenting{" "}
+							<span className="py-[0px] px-[4px] text-black--30 bg-black-70 rounded-[2px]">
+								/giselle {callSign === "" ? "[call sign]" : callSign}
+							</span>{" "}
+							in the issue route06inc/giselle.
```

Comment:
This should be replaced with `repositoryFullName`?

Reply:
@satococoa That's right. However, dynamically changing the content here is a bit of a change, so we will use a separate Pull reqesut.
It will be hidden in this Pull request.

b192336b7335dbbf10b0022522c915c2d6a104e3